python/lsst/verify/report.py

```diff
@@ -87,7 +88,11 @@ def make_table(self):
 
             spec = self._spec_set[spec_name]
 
-            if spec.check(meas.quantity):
+            if np.isnan(meas.quantity):
```

Comment:
I'm not 100% sure `numpy.isnan` is what we want here.

What sets a measurement's `quantity` attribute to `np.nan`? I know the default value is `None` in `Measurement`.

Reply:
After talking it through this seems right. :+1: